app/src/main/java/MainMenu.java

```diff
@@ -0,0 +1,116 @@
+import static com.raylib.Jaylib.*;
+
+import com.raylib.Jaylib;
+
+import java.util.ArrayList;
+
+/** The MainMenu class manages the game's main menu options using the Singleton Pattern. */
+public class MainMenu {
+
+    // Instance variable for the Singleton Pattern
+    private static final MainMenu instance = new MainMenu();
+
+    private Util.Background background;
+    private ArrayList<UtilButton> button;
```

Comment:
Es wäre übersichtlicher wenn alle Buttons einzeln gespeichert werden anstatt in einer Arraylist also singleplayerButton, mutliplayerButton und exitButton